src/main/apiClient.ts

```diff
@@ -484,6 +485,16 @@ export default class ApiClient {
     })
   }
 
+  sendVerificationEmail(params: EmailVerificationParams): Promise<void> {
+    const { accessToken, ...data } = params
+    return this.http.post('/send-email-verification', { body: { ...data }, accessToken })
+  }
+
+  sendVerificationSms(params: { accessToken: string }): Promise<void> {
```

Comment:
I would rename the method `sendVerificationPhoneNumber` to be uniform with the other methods names (`updatePhoneNumber`, `verifyPhoneNumber`).

Reply:
I decided to use SMS instead of PhoneNumber in this case because as a phrase `sendVerificationSms` reads better and means : 
>Send a verification SMS.

I guess I should revert to my first method name : `sendPhoneNumberVerification`. Would that be alright?

Reply:
Yes, I understand your choice. But I find it easier to distinguish if the method is for email or phone number when the identifier is at the end of the word. However, this is my own opinion. Since it's not important, do as you want 😄

Reply:
Do you think I should change the UI label also from SMS to Phone Number ?

Reply:
On the Sandbox?

Reply:
@FrancoisDevemy5  Sorry if this was already answered I didn't follow the whole discussion but the name of the new methods was specified in the Asana tickets: **sendEmailVerification** and **sendPhoneNumberVerification** to be coherent with the endpoint.

Reply:
Ok.